packages/pie-toolbox/src/code/editable-html/plugins/toolbar/editor-and-toolbar.jsx

```diff
@@ -85,7 +87,7 @@ export class EditorAndToolbar extends React.Component {
       <div
         className={classNames(
           {
-            [classes.noBorder]: toolbarOpts && toolbarOpts.noBorder,
+            [classes.noBorder]: toolbarOpts && toolbarOpts.noBorder || noBorder,
```

Comment:
Let's refactor this, and only use toolbarOpts. I understand we have the noPadding situation, but this might be the right time to fix that.